docs/about/cookies.md

```diff
@@ -1,13 +1,17 @@
 ---
 title: Cookie Statement
 description: Explanation of Able Player's use of functional cookies
+layout: default
+parent: About this website
+nav_order: 3
 ---
 
-## 🍪 Cookie Statement
+# Cookie Statement
 
-Able Player uses cookies to enhance accessibility and improve user experience. These cookies are **strictly functional**—they help remember your preferences, such as playback settings, but do **not** track your activity or collect personal data.
+Able Player uses cookies to enhance accessibility and improve user experience. These cookies are **strictly functional**—they help remember your preferences, such as playback settings, but do **not** track your activity or collect personal data. 
+No other cookies are used except for the cookies the Able Player sets.
 
-### 🔧 What kind of cookies are used?
+## What kind of cookies are used?
 
 Able Player uses the [js-cookie](https://github.com/js-cookie/js-cookie) library to store and retrieve user preferences. These include:
```

Comment:
This section should include information about the type of cookie: Essential, Performance, Functional, or Marketing. The Able Player cookies are classified as "Functional" - cookies to provide enhanced, personalized features such as remembering user choices and preferences.